domains/POAS/events/reduced-fee-approved/examples/example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "uid": "M-1234-5678-9012",
+  "outstandingFee": 4100
```

Comment:
Could it be called `fee` or even `feePence`? The "outstanding" makes it sound like it is calculating what is left to pay

Reply:
Ah sorry that's what I thought we wanted but I must have misunderstood. So do want the amount to be the new expected total to pay? E.g A remission approved is £41, but a remission or exemption declined is £82?

Reply:
yeah that will be simpler, we have the amount in MRLPA of what has been paid so can then do `fee - paid = outstanding`